cdap-docs/developers-manual/source/building-blocks/workflows.rst

```diff
@@ -68,14 +83,111 @@ custom action, you will need to extend the ``AbstractWorkflowAction`` and implem
     }
   }
 
-The custom action then can be added to the Workflow using the ``addAction()`` method as shown above.
+The custom action then can be added to the Workflow using the ``addAction()`` method as
+shown above.
+
+Concurrent Workflows
+--------------------
+By default, a Workflow runs sequentially. Multiple instances of a Workflow can be run
+concurrently. To enable concurrent runs for a Workflow, set its runtime argument
+``concurrent.runs.enabled`` to ``true``.
+
+.. _workflow_parallel:
+
+Parallel Workflows
```

Comment:
Should we call it ``Parallelizing Workflow execution``. The reason is ``Parallel Workflows`` makes it feel like we are executing the entire Workflow in parallel (which would be similar to the Concurrent Workflow above).